_config.yml

```diff
@@ -72,7 +72,16 @@ authors:
     twitter: hauspoor
     about:  "Elliott Hauser is an Assistant Professor at the UT Austin iSchool.  He's hacking education as one of the cofounders of Trinket.io."
 
-
+authors:
```

Comment:
Hi!  Here is where I have changed things, I hope I understood the assignment correctly!  I'm sorry to miss class today, I'll see you next week and review all the materials before then.  Best, Hannah